platform/jvm/capture/src/test/kotlin/io/bitdrift/capture/AppExitLoggerTest.kt

```diff
@@ -38,25 +40,28 @@ class AppExitLoggerTest {
     private val logger: LoggerImpl = mock()
     private val activityManager: ActivityManager = mock()
     private val runtime: Runtime = mock()
+
     private val errorHandler: ErrorHandler = mock()
     private val crashHandler: CaptureUncaughtExceptionHandler = mock()
     private val versionChecker: BuildVersionChecker = mock()
     private val memoryMetricsProvider = FakeIMemoryMetricsProvider()
-    private val appExitLogger =
-        AppExitLogger(
-            logger,
-            activityManager,
-            runtime,
-            errorHandler,
-            crashHandler,
-            versionChecker,
-            memoryMetricsProvider,
-        )
+    private lateinit var appExitLogger: AppExitLogger
 
     @Before
     fun setUp() {
+        CaptureDispatchers.setTestExecutorService(MoreExecutors.newDirectExecutorService())
```

Comment:
this seems to be a roundabout way to inject a test injector? can't you just pass `MoreExecutors.newDirectExecutorService()` as the last param of `AppExitLogger()` ctor? that's how we do it in all of our other tests